From c3c4fa01be2c96a78d2ad08f13a8b3330e7bf168 Mon Sep 17 00:00:00 2001
From: Frank Wang <frank.wang@rock-chips.com>
Date: Tue, 21 Nov 2017 17:29:07 +0800
Subject: [PATCH] ARM64: configs: add usb related config to rockchip_defconfig

Enable CONFIG_USB_DWC2 and CONFIG_PHY_ROCKCHIP_INNO_USB3 driver
for rockchip_defconfig.

Change-Id: Ic22cc450e5f139935dc54ffb798413edff7df3cb
Signed-off-by: Frank Wang <frank.wang@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 4709d7ccb4a4..cd6462850c6d 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -634,6 +634,7 @@ CONFIG_USB_STORAGE_ENE_UB6250=y
 CONFIG_USB_UAS=y
 CONFIG_USB_DWC3=y
 # CONFIG_USB_DWC3_PCI is not set
+CONFIG_USB_DWC2=y
 CONFIG_USB_SERIAL=y
 CONFIG_USB_SERIAL_GENERIC=y
 CONFIG_USB_SERIAL_OPTION=y
@@ -718,6 +719,7 @@ CONFIG_PWM=y
 CONFIG_PWM_ROCKCHIP=y
 CONFIG_PHY_ROCKCHIP_USB=y
 CONFIG_PHY_ROCKCHIP_INNO_USB2=y
+CONFIG_PHY_ROCKCHIP_INNO_USB3=y
 CONFIG_PHY_ROCKCHIP_EMMC=y
 CONFIG_PHY_ROCKCHIP_DP=y
 CONFIG_PHY_ROCKCHIP_INNO_MIPI_DPHY=y
-- 
2.35.3

